src/gui/properties/speedplotview.h

```diff
@@ -86,6 +90,19 @@ class SpeedPlotView : public QGraphicsView
     void paintEvent(QPaintEvent *event) override;
 
 private:
+    class Averager
+    {
+    public:
+        Averager(int divider, boost::circular_buffer<PointData> &sink);
+        inline void push(const PointData &p);
```

Comment:
inline is meaningless here. Please remove it.

Reply:
Removed